powerlevel9k.zsh-theme

```diff
@@ -1410,20 +1410,22 @@ prompt_dir_writable() {
   fi
 }
 
-# Kubernetes Current Context
+# Kubernetes Current Context/Namespace
 prompt_kubecontext() {
   local kubectl_version="$(kubectl version --client 2>/dev/null)"
 
   if [[ -n "$kubectl_version" ]]; then
-    # Get the current Kubernetes config context's namespaece
-    local k8s_namespace=$(kubectl config get-contexts --no-headers | grep '*' | awk '{print $5}')
+    local cur_ctx=$(kubectl config view -o=jsonpath='{.current-context}')
+    cur_namespace="$(kubectl config view -o=jsonpath="{.contexts[?(@.name==\"${cur_ctx}\")].context.namespace}")"
```

Comment:
Do you know what version of Kubectl they added support for -o=jsonpath?

Reply:
Hmmm.. The earliest official version on kubernetes.io is v1.5 and it supports `jsonpath` flag --> [kubectl config view](https://kubernetes.io/docs/user-guide/kubectl/v1.5/#-em-view-em-).

Reply:
Thought it might be important to highlight that.